package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentful-ui-extensions-sdk",
   "description": "SDK to develop custom UI Extension for the Contentful Web App",
-  "version": "3.16.1",
+  "version": "3.16.2",
```

Comment:
It's a bit hard to glean but is this really a patch? Will users be able to bump this without any code change?